codec/test_codec.go

```diff
@@ -143,7 +143,7 @@ func TestStruct(codec GeneralCodec, t testing.TB) {
 
 	bytesLen, err := manager.Size(0, myStructInstance)
 	require.NoError(err)
-	require.Equal(len(myStructBytes), bytesLen)
+	require.Len(myStructBytes, bytesLen)
```

Comment:
These are kind of interesting... Here, we are testing the `bytesLen` value, not the `myStructBytes` value... I think we should only be banning the usage of `len` as the second variable in `Equal` (and that these should still use `Equal`)